lib/integrations/awsoidc/deployservice_test.go

```diff
@@ -19,14 +19,19 @@
 package awsoidc
 
 import (
+	"context"
 	"regexp"
 	"testing"
 
+	"github.com/aws/aws-sdk-go-v2/aws"
+	ecsTypes "github.com/aws/aws-sdk-go-v2/service/ecs/types"
```

Comment:
```suggestion
	ecstypes "github.com/aws/aws-sdk-go-v2/service/ecs/types"
```
Package names (even aliased ones) should use all lower case.